coucharchive

```diff
@@ -172,6 +172,7 @@ class CouchDBInstance(object):
                     'database_dir = %s\n' % self.datadir +
                     'view_index_dir = %s\n' % self.datadir +
                     'max_dbs_open = 10000\n'
+                    'users_db_security_editable = true\n'  # for CouchDB 3+
```

Comment:
When using CouchDB v2, the parameters is just ignore is that it? (If yes great!)

Reply:
Yes sir! :slightly_smiling_face:

Reply:
Nice, thanks!